tone-analyzer/src/main/java/com/ibm/watson/developer_cloud/tone_analyzer/v3/model/UtterancesTone.java

```diff
@@ -0,0 +1,34 @@
+package com.ibm.watson.developer_cloud.tone_analyzer.v3.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Main object containing the result of running the Tone Analyzer Chat Analysis on a conversation. It contains a list
+ * of utterance-level analysis results.
+ */
+public class UtterancesTone extends GenericModel {
+
+  @SerializedName("utterancesTone")
+  List<UtteranceAnalysis> utterancesTone = new ArrayList<>();
```

Comment:
The API has `utterances_tone` instead of `utterancesTone`, but this fails the checkstyle step. Changing the API itself might take some time, can I let this be `utterances_tone`?